controllers/teams/updater.js

```diff
@@ -36,17 +36,14 @@ const update = async (req, res) => {
       where: { id: teamId },
       fields: allowedFields 
     });
-    res.status(200).json({
-      success: true,
-      message: `Modified team ${teamId}`
-    });
+    responses.singleDTOResponse(res,200,"Equipo modificado con exito del id: ",teamId);
   } catch (error) {
     console.error(error);
     if (typeof req.file !== 'undefined') {
       fileController.deleteFile(req.file.path, req.file.filename);
-      res.status(400).send(error.message);
+      responses.errorDTOResponse(res,400,error.message);
     } else {
-      res.status(400).send(error.message + '\nError: img not sent');
+      responses.errorDTOResponse(res,400,error.message + '\nError: imagen no se subio correctamente o no se envio');
```

Comment:
```suggestion
    if (typeof req.file !== 'undefined') {
      fileController.deleteFile(req.file.path, req.file.filename);
      return responses.errorDTOResponse(res, 400, error.message);
    }
    return responses.errorDTOResponse(res, 500, "La imagen enviada no se pudo procesar adecuadamente o no se envió una imagen");
```
Nuevamente, se aceptan mejores correcciones para que la guard clausure se vea más bomnita @Hector1567XD